### PR TITLE
Minor fixes in docstring of keras Model.compile method

### PR DESCRIPTION
**Pull request by j-i-l**
- Consistent usage of backquotes
- minor change in wording
- misspelling ie. > i.e.

### CODE REVIEW

**Review by deeb02 — COMMENTED** [review]
Looks like a big improvement. Left one minor comment.

File: tensorflow/python/keras/engine/training.py [hunk 1]
```diff
@@ -501,12 +501,13 @@ def compile(self,
...
-          used and reduction is set to NONE, return value has the shape
-          [batch_size, d0, .. dN-1] ie. per-sample or per-timestep loss values;
+          used and reduction is set to `None`, return value has the shape
+          `[batch_size, d0, .. dN-1]` i.e. per-sample or per-timestep loss values;
           otherwise, it is a scalar. If the model has multiple outputs, you can
           use a different loss on each output by passing a dictionary or a list
           of losses. The loss value that will be minimized by the model will
-          then be the sum of all individual losses.
+          then be the sum of all individual losses, unless `loss_weights` is 
+          specified.
```
[the remark this answers is not in the document]
Comment (j-i-l) [reply tensorflow/python/keras/engine/training.py:510]: I would concentrate information about a parameter at a single point, whenever possible, and thus avoid putting any description of the `loss_weights` parameter in the description of `loss`.

This reduces redundancy in the documentation and helps keeping it short and concise. It also makes the maintenance of the documentation easier.

[the remark this answers is not in the document]
Comment (gbaned) [reply tensorflow/python/keras/engine/training.py:510]: @deeb02 Can you please take a look on the above comment from @j-i-l. Thanks!

[the remark this answers is not in the document]
Comment (deeb02) [reply tensorflow/python/keras/engine/training.py:510]: Honestly, not sure I follow the argument, but it's a minor point. LGTM.

**Review by deeb02 — APPROVED** [review]
Looks good to me.

Thanks for the improvement.

**Comment by rthadur** [comment]
@j-i-l please check sanity build failures ?

**Comment by j-i-l** [comment]
@rthadur, I fixed the pylint errors (line-too-long) in cd0a3b6

**Review by deeb02 — APPROVED** [review]
Looks good to me.
Thanks for the improvement.
